streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -1053,7 +1054,7 @@ int maybeCommitActiveTasksPerUserRequested() {
     }
 
     private void commitOffsetsOrTransaction(final Map<Task, Map<TopicPartition, OffsetAndMetadata>> offsetsPerTask) {
-        log.debug("Committing task offsets {}", offsetsPerTask);
+        log.debug("Committing task offsets {}", offsetsPerTask.entrySet().stream().collect(Collectors.toMap(t -> t.getKey().id(), Entry::getValue))); // avoid logging actual Task objects
```

Comment:
An unrelated but equally annoying thing I noticed in the logs: we should never log a full `Task` object because it prints literally everything about the task, including for example the topology description which is not that useful but sometimes VERY long